apps/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,38 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { CardComponent } from '../../ui/card/card.component';
+import { City } from '../../model/city.model';
+import { CardType } from '../../model/card.model';
+import { FakeHttpService } from '../../data-access/fake-http.service';
+import { CityStore } from '../../data-access/city.store';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: ` <app-card [list]="cities" [type]="cardType"> </app-card>`,
   standalone: true,
-  imports: [],
+  imports: [CardComponent],
+  styles: [
+    `
+      app-city-card div {
+        background-color: orange;
+      }
+    `,
+  ],
+  encapsulation: ViewEncapsulation.None,
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cities: City[] = [];
+  cardType = CardType.CITY;
 
-  ngOnInit(): void {}
+  constructor(private http: FakeHttpService, private store: CityStore) {}
+
+  ngOnInit(): void {
+    this.fetchCities();
+  }
+
+  fetchCities(): void {
+    this.http.fetchCities$.subscribe((cities: City[]) =>
+      this.store.addAll(cities)
+    );
+    this.store.cities$.subscribe((cities: City[]) => (this.cities = cities));
```

Comment:
you should try to avoid to use the suscribe function. Try to always use asyncpipe or use a componentStore from ngrx or rxstore from rxangular. They will handle the subciption for you otherwise don't forget to unsuscribe.

And using subscribe will create imperative code which is less maintanable. Using asyncpipe or reactive pattern will create declarative code which is easier to understand and maintain since each variable are created at a unique place.